Sources/AppKitBackend/AppKitBackend.swift

```diff
@@ -1724,6 +1758,60 @@ final class NSCustomTapGestureTarget: NSView {
     }
 }
 
+final class NSCustomHoverTarget: NSView {
+    var hoverChangesHandler: ((Bool) -> Void)? {
+        didSet {
+            if hoverChangesHandler != nil && trackingArea == nil {
+                let options: NSTrackingArea.Options = [
+                    .mouseEnteredAndExited,
+                    .activeInKeyWindow,
+                ]
+                let area = NSTrackingArea(
+                    rect: self.bounds,
+                    options: options,
+                    owner: self,
+                    userInfo: nil)
+                addTrackingArea(area)
+                trackingArea = area
+            } else if hoverChangesHandler == nil, let trackingArea {
+                // should be impossible at the moment of implementation
+                // keeping it to be save in case of later changes
+                removeTrackingArea(trackingArea)
+                self.trackingArea = nil
+            }
+        }
+    }
+
+    private var trackingArea: NSTrackingArea?
+
+    override func updateTrackingAreas() {
+        super.updateTrackingAreas()
+        if let trackingArea = trackingArea {
+            self.removeTrackingArea(trackingArea)
+        }
+        let options: NSTrackingArea.Options = [
+            .mouseEnteredAndExited,
+            .activeInKeyWindow,
+        ]
+
+        trackingArea = NSTrackingArea(
+            rect: self.bounds,
+            options: options,
+            owner: self,
+            userInfo: nil)
+        self.addTrackingArea(trackingArea!)
```

Comment:
Avoid the force unwrap with an intermediate variable like you did in the didSet above.